style.css

```diff
@@ -0,0 +1,17 @@
+img {
+    max-width: 100%;
+    display: block;
+    height: auto;
+    margin: 0 auto;
+  }
+  .title_paragraph{
+    display: block;
+    text-align: center;
+    background-color: rgb(233, 225, 225);
+  }
+  #tribute-info{
+    display: block;
+    text-align: center;
+    background-color: rgb(233, 225, 225);;
+  }
+  
```

Comment:
Instead of display block i suggest you to use display flex, because display flex makes your page responsive.